packages/beacon-node/src/monitoring/properties.ts

```diff
@@ -1,24 +1,24 @@
 import {Registry} from "prom-client";
 import {JsonRecord, JsonType, MetricObject, MetricValue, MetricWithGetter, RecordValue} from "./types.js";
 
-interface PropertyDefinition {
+type PropertyDefinition = {
```

Comment:
This change is not really related to the PR, I want to sneak it in tho :)

see https://github.com/ChainSafe/lodestar/pull/5187#discussion_r1116167127 on why this is changed